stable/jenkins/values.yaml

```diff
@@ -504,6 +504,18 @@ networkPolicy:
   # For Kubernetes v1.4, v1.5 and v1.6, use 'extensions/v1beta1'
   # For Kubernetes v1.7, use 'networking.k8s.io/v1'
   apiVersion: networking.k8s.io/v1
+  # You can allow agents to connect from both within the cluster (from within specific/all namespaces) AND/OR from a given external IP range
+  # whether agents are allowed to connect from within the cluster and from which namespaces.
+  internalAgents:
+    allowed: true
+  # uncomment the lines below to allow agents to connect ONLY from namespaces with the defined labels
+  #  namespaceLabels:
+  #    project: myproject
+  # uncomment the lines below to allow agents from outside the cluster and from a given IP range to connect to master
+  # externalAgents:
+  #   ipCIDR: 172.17.0.0/16
+  #   except:
+  #     - 172.17.1.0/24
```

Comment:
I suggest to remove the comment from the keys (namespaceLabels and externalAgents) and set it to empty.
The examples can be commented.

Having the keys without a comment allows IDEs to do auto completion for values.

Reply:
Thanks for the tip! I have modified the values.